src/ontology/mondo-ingest.Makefile

```diff
@@ -199,6 +199,14 @@ $(REPORTDIR)/%_mapping_status.tsv $(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR
 	--outpath-simple $(REPORTDIR)/$*_unmapped_terms.tsv \
 	--outpath-full $(REPORTDIR)/$*_mapping_status.tsv
 
+components/%-unmapped.owl: components/%.owl reports/%_unmapped_terms.tsv
```

Comment:
@matentzn All prereqs keep re-running each time.
I'm not sure why, but even with `IMP=false COMP=false MIR=false`, this goal still re-runs the entire prerequisite tree even if all of the artefacts already exist. I looked a the terminal output, and I see that `if [ false = true ] && [ false = true ]` is occurring where I would expect in the component goal. I don't see `-B` anywhere in any of these prerequisites. I could look a bit harder, but at the moment I'm not sure why it seems like everything is running.

Reply:
Which is the first goal that is being re-run? So when you run a target, which is the very first goal that is rerun although it shouldn't?

Reply:
Good suggestion. I took a look, and it's: `reports/mirror_signature-%.tsv`

It depends on `component-download-%.owl`. Hmm... Maybe the issue here is that these component download goals are phony. Is this what triggers the whole pipeline to run? 

If this is the case, then even my suggestion with the MD5 stuff won't fix it. Is there any way that we can make these not phony?

Reply:
There is a bug somewhere I think, I noticed that as well. This is how it should work:

.PHONY goals for download always get triggered. They then create the downloaded file somewhere, but _only if it has changed_. Example:

https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/ontology/Makefile#L423

Note that goal that creates the file works like this: https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/ontology/Makefile#L567

So, `myfile.owl` depends on `mirror/chebi.owl` (as its only dependency). mirror/chebi.owl triggers the phony `mirror-chebi`. This may or may not change `mirror/chebi.owl` (using file equivalence as a proxy). If `mirror/chebi.owl`, `myfile.owl` will not be rebuild either. Somewhere this logic is being disrupted..

Reply:
Ah, I looked at the code and your explanation and I see how this works. I'll compare to what's going on here briefly with `mondo-ingest` and see if I can easily find a difference.